py4DSTEM/process/dpc/dpc.py

```diff
@@ -209,7 +209,7 @@ def get_rotation_and_flip_maxcontrast(CoMx, CoMy, N_thetas, paddingfactor=2,
                                           n_iter=n_iter)
         stds[i] = np.std(phase)
         if verbose:
```

Comment:
if verbose = True results in pass, should it be stripped out, or is the plan to restore a progress bar?

Reply:
I replaced it with tqdm